.github/workflows/build-multi-arch.yml

```diff
@@ -0,0 +1,42 @@
+name: Build x64 and ARM64 images 
+
+on:
+  pull_request:
+    branches:
+      - 'master'
+      - 'release*'  
+
+jobs:
+  build_x64:
+    name: Build x64
+    runs-on: [self-hosted, linux, x64]
+    steps:
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: '1.14.3'
```

Comment:
In our make file , we build with go 1.16 

https://github.com/aws/amazon-vpc-cni-k8s/blob/master/Makefile#L64 

Does the worker version go-1.14 have any significance or relevance? Because we do `make docker ...` the targets below.

Reply:
I don't think there is any relevance. It is mainly to pull the code into go directory.